pytorch_toolkit/object_detection/model_templates/custom-detection/tools/eval.py

```diff
@@ -17,8 +17,8 @@
 
 from ote import MODEL_TEMPLATE_FILENAME
 from ote.api import test_args_parser
-from oteod.args_conversion import convert_ote_to_oteod_test_args
-from oteod.evaluation.common import evaluate
+from ote.args_conversion import convert_ote_to_oteod_test_args
```

Comment:
The oteod name does not exist anymore, I would propose to rename it.

Reply:
`oteod` in names were removed.